docs/aaaCtl/Account-Dashboard.md

```diff
@@ -0,0 +1,34 @@
+# Account Dashboard
+
+To view the FullCtl Account Dashboard (aka aaaCtl), go to: account.fullctl.io, enter username (or email address)  and password and click Login. 
```

Comment:
account.fullctl.io should be a link here I think, please use the `[account.fullctl.io](https://account.fullctl.io)` syntax